applications/GeoMechanicsApplication/custom_processes/apply_boundary_phreatic_line_pressure_table_process.hpp

```diff
@@ -31,65 +30,41 @@ class ApplyBoundaryPhreaticLinePressureTableProcess : public ApplyConstantBounda
     /// Defining a table with double argument and result type as table type.
     typedef Table<double,double> TableType;
 
-///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-    /// Constructor
     ApplyBoundaryPhreaticLinePressureTableProcess(ModelPart& model_part,
                                                  Parameters rParameters
                                                  ) : ApplyConstantBoundaryPhreaticLinePressureProcess(model_part, rParameters)
     {
         KRATOS_TRY
 
         unsigned int TableId = rParameters["table"].GetInt();
-        mpTable = model_part.pGetTable(TableId);
+        mpTable            = model_part.pGetTable(TableId);
         mTimeUnitConverter = model_part.GetProcessInfo()[TIME_UNIT_CONVERTER];
 
         KRATOS_CATCH("")
     }
 
-    ///------------------------------------------------------------------------------------
-
-    /// Destructor
-    ~ApplyBoundaryPhreaticLinePressureTableProcess() override {}
-
-///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-    /// Execute method is used to execute the ApplyBoundaryPhreaticLinePressureTableProcess algorithms.
-    void Execute() override
-    {
-    }
+    ApplyBoundaryPhreaticLinePressureTableProcess(ApplyBoundaryPhreaticLinePressureTableProcess const& rOther) = delete;
+    ApplyBoundaryPhreaticLinePressureTableProcess& operator=(ApplyBoundaryPhreaticLinePressureTableProcess const& rOther) = delete;
```

Comment:
In line with my first comment:

```suggestion
    ApplyBoundaryPhreaticLinePressureTableProcess(const ApplyBoundaryPhreaticLinePressureTableProcess&) = delete;
    ApplyBoundaryPhreaticLinePressureTableProcess& operator=(const ApplyBoundaryPhreaticLinePressureTableProcess&) = delete;
```

This suggestion may apply to other classes too (I won't repeat them).

Reply:
Done in all places I could find in custom_processes.